internal/utils/copy_from_image.go

```diff
@@ -38,6 +39,56 @@ func CopyFromImage(image, srcInImage, destOnHost string) error {
 	if err != nil {
 		return fmt.Errorf("copy from image: %s: %w", stderr.String(), err)
 	}
+	excludeFiles := []string{".gitkeep", ".ini", ".env", ".DS_Store"}
```

Comment:
What is `.ini`? I guess you want match any file ended with `.ini`, not a file with `.ini` filename.

Reply:
.ini means a hiden file whose filename is `.ini`, this file always contains sensitive data.